### PR TITLE
Bump Check version

### PR DESCRIPTION
**Pull request by bramtayl**

### CODE REVIEW

File: C/Check/build_tarballs.jl [hunk 1]
```diff
@@ -3,17 +3,17 @@
 using BinaryBuilder, Pkg
 
 name = "Check"
-version = v"0.15.1"
+version = v"0.15.2"
 
 # Collection of sources required to complete build
 sources = [
-    ArchiveSource("https://github.com/libcheck/check/releases/download/0.15.1/check-0.15.1.tar.gz", "c1cc3d64975c0edd8042ab90d881662f1571278f8ea79d8e3c2cc877dac60001")
+    ArchiveSource("https://github.com/libcheck/check/releases/download/0.15.2/check-0.15.2.tar.gz", "a8de4e0bacfb4d76dd1c618ded263523b53b85d92a146d8835eb1a52932fa20a")
```
Comment (giordano) [inline C/Check/build_tarballs.jl:10]:
```suggestion
    ArchiveSource("https://github.com/libcheck/check/releases/download/$(version)/check-$(version).tar.gz", "a8de4e0bacfb4d76dd1c618ded263523b53b85d92a146d8835eb1a52932fa20a")
```

File: C/Check/build_tarballs.jl [hunk 2]
```diff
@@ -3,17 +3,17 @@
...
 ]
 
 # Bash recipe for building across all platforms
 script = raw"""
 cd $WORKSPACE/srcdir
-cd check-0.15.1
+cd check-0.15.2
```
Comment (giordano) [inline C/Check/build_tarballs.jl:16]:
```suggestion
cd check-*
```

**Comment by bramtayl** [comment]
@giordano this doesn't seem to have triggered a new jll? No problem if it just takes a couple days, just wanted to make sure nothing went wrong?

**Comment by giordano** [comment]
It looks like registration failed, no idea why: https://dev.azure.com/JuliaPackaging/Yggdrasil/_build/results?buildId=8566&view=logs&j=37c85844-67e3-524d-1549-527eae771711&t=1fdc72a5-23ec-514b-378b-07bf49fec5cc.  Could you please edit the file (any dummy change) and open a new PR?

**Pull request merged**